userspace/falco/falco.cpp

```diff
@@ -462,6 +470,8 @@ int falco_init(int argc, char **argv)
         {"mesos-api", required_argument, 0, 'm'},
         {"option", required_argument, 0, 'o'},
         {"pidfile", required_argument, 0, 'P'},
+	{"psp", required_argument, 0},
```

Comment:
Indentation

Reply:
I ran everything through clang-format using the existing https://github.com/falcosecurity/falco/blob/dev/.clang-format